test/config/ra.json

```diff
@@ -41,10 +42,12 @@
       "address": ":9094",
       "clientNames": [
         "wfe.boulder",
-        "admin-revoker.boulder"
+        "admin-revoker.boulder",
+        "bad-key-revoker.boulder"
       ]
     },
     "features": {
+      "StoreRevokerInfo": true
```

Comment:
Looks like the FasterNewOrdersRateLimit db migration hasn't been pulled into sa/_db, but I'm not sure what else RestrictRSAKeySizes is tied to. Why isn't that one being brought over?

Reply:
We aren't planning to turn on RestrictRSAKeySizes till we've had a chance to analyze current key distributions and notify people who are using uncommon sizes.